Assembler.Core/Entities/BaseMessageInAssembly.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Assembler.Core.Enums;
+
+namespace Assembler.Core.Entities
+{
+    public abstract class BaseMessageInAssembly : BaseMessage
+    {
+        public ReleaseReason ReleaseReason;
+        public bool MiddleReceived;
```

Comment:
In both lines above: why no `{ get; set; }` (or just `{ get; }`)?

Reply:
Is there a difference?

Reply:
1. The compiler treats it differently (supposedly it could cause errors in some cases)
2. I see this as a massive conventions, in every C# code I look through. If there's some reason not to do so, I'd like to at least know why

Reply:
No reason not to use, I just don't see why.
I've added it.

Reply:
👍